doc/plugin.rst

```diff
@@ -0,0 +1,108 @@
+##############
+Spyder plugins
+##############
+
+Spyder is made up of multiple plugins that perform a specific task, 
+for instance spyder-terminal or the variable explorer. As well, plugins are
```

Comment:
Please add a link to `spyder-terminal` and to the page in our docs that describes the Variable Explorer.

Besides, please capitalize `variable explorer`.